Riot/Modules/Threads/ThreadsCoordinator.swift

```diff
@@ -68,7 +68,7 @@ final class ThreadsCoordinator: NSObject, ThreadsCoordinatorProtocol {
         // Detect when view controller has been dismissed by gesture when presented modally (not in full screen).
         self.navigationRouter.toPresentable().presentationController?.delegate = self
         
-        guard parameters.threadId == nil else {
+        guard parameters.threadId != nil else {
```

Comment:
I can't remember what this was doing but can you explain this change a bit?

Reply:
I removed the guard as we need the pop completion block to be triggered for all cases. RoomViewControllers where not released when popped and RR where still sent even if the messages were not read.